global.json

```diff
@@ -11,7 +11,7 @@
     }
   },
   "msbuild-sdks": {
-    "Microsoft.DotNet.Arcade.Sdk": "1.0.0-beta.19458.2",
-    "Microsoft.DotNet.Helix.Sdk": "2.0.0-beta.19458.2"
+    "Microsoft.DotNet.Arcade.Sdk": "4.0.0-dev",
+    "Microsoft.DotNet.Helix.Sdk": "2.0.0-beta.19454.31"
```

Comment:
Do not change this portion of the file directly. Need to update the dependencies from Arcade i.e. get `darc` to change Versions.props and Version.Details.xml as well.

In your branch, execute
``` powershell
darc update-dependencies --source-repo arcade  --channel '.NET Tools - Latest'
```
or
``` powershell
darc update-dependencies --source-repo arcade  --channel '.NET 3 Tools'
```